client/lib/feed-post-store/post-batch-fetcher.js

```diff
@@ -88,8 +88,8 @@ assign( PostBatchFetcher.prototype, {
 					return;
 				}
 
-				containerId = +match[ 1 ];
-				postId = +match[ 2 ];
+				containerId = match[ 1 ];
```

Comment:
because we use it as a key into a map later on. I'd rather not mix numeric and string keys.